pkg/pool-manager/pool_test.go

```diff
@@ -414,4 +416,61 @@ var _ = Describe("Pool", func() {
 		})
 	})
 
+	Describe("Multus Network Annotations API Tests", func() {
+		pod := v1.Pod{ObjectMeta: metav1.ObjectMeta{Name: "testPod", Namespace: "default"}}
+		poolManager := &PoolManager{}
```

Comment:
I'm sorry, I commented this should be in `Describe`, while it should be in the `Context` where we set it in `BeforeEach`.